designer/server/src/test/scala/pl/touk/nussknacker/ui/validation/UIProcessValidatorSpec.scala

```diff
@@ -1270,6 +1278,168 @@ class UIProcessValidatorSpec extends AnyFunSuite with Matchers with TableDrivenP
     result.warnings shouldBe ValidationWarnings.success
   }
 
+  test(
+    "validate Map service parameter based on additional config from provider - ValidationExpressionParameterValidator"
+  ) {
+    val process = createGraph(
+      List(
+        Source("inID", SourceRef(ProcessTestData.existingSourceFactory, List())),
+        Enricher(
+          "custom",
+          ServiceRef(
+            "mapParameterService",
+            List(
+              NodeParameter(
+                ParameterName("mapParam1"),
+                Expression.spel("""{'field': "valid", 'otherField': 'c'}""")
```

Comment:
is the 'otherField' important to test? cant it just be `{'field': 'valid'}`?

Reply:
it's not, i'll remove it